dist/index.html

```diff
@@ -4,31 +4,92 @@
     <meta charset="utf-8">
     <title>Seek-A-Design Playground</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7fcb54df80.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
     <script src="../src/index.js"></script>
   </head>
   <body>
     <div class="parent-container">
-      <!-- <h1 class="center">Hello World!</h1> -->
       <div class="navigation-container">
-        <button type="button" class="nav">Home</button>
-        <button type="button" class="nav">Projects</button>
-        <button type="button" class="nav">About Me</button>
-        <button type="button" class="nav">Contact</button>
+        <button type="button" class="nav-home">
+            <i class="fas fa-home"></i> Home
+        </button>
+        <button type="button" class="nav-projects">
+            <i class="fas fa-project-diagram"></i> Projects
+        </button>
+        <button type="button" class="nav-about">
+            <i class="fas fa-info-circle"></i> About Me
+        </button>
+        <button type="button" class="nav-contact">
+            <i class="fas fa-comment-dots"></i> Contact</button>
+        <div class="lower-nav">Custom Designs · Marketing · Branding · And More</div>
       </div>
-      <span class="perks">
-        Custom Designs · Marketing · Branding · And More
-      </span>
-      <div class="whois">
-        <h2> WHO IS SEEK-A-DESIGN?</h2>
-        </br>
-        <p>Seek-A-Design was a long-time dream of owner and designer, Lyandra Sica (seek•uh). Using her last name, she developed the play-on-words into her business which has sprouted vivdly within the last few years. Within those years, her hard work and determination has allowed her dream to turn into the reality of being a go-to designer and printing expert.</p>
+
+      <div class="whoisseeking-container">
+        <div class="whois">
+          <h2> WHO IS SEEK-A-DESIGN?</h2>
+          <hr>
+          <div class="whois-text">
+            <p>
+              Seek-A-Design was a long-time dream of owner and designer, Lyandra Sica (seek•uh). Using her last name, she developed the play-on-words into her business which has sprouted vivdly within the last few years. Within those years, her hard work and determination has allowed her dream to turn into the reality of being a go-to designer and printing expert.
```

Comment:
*Lines 34, 38, and 42:*

Can we wrap our text here, keeping it less than 80 characters wide? 
(Use the vertical line in your editor to easily identify the 80 character wide mark.)

Reply:
I have  the 80 character line in atom now. Is there any sort of wrapping package available that will drop the text to the next line one it reaches the 80th character, that you know of?

Reply:
I'm sure there is, but it is best to just hit enter and do it manually yourself, otherwise there could be errors. Let's just stick with that for now. I have had bad experience with text-wrap in editors in the past.